### PR TITLE
Cleanup for wrapped assets

### PR DESCRIPTION
**Pull request by diego-G**
- [x] Remove CMTAT standalone
- [x] Remove Enforcement mandatory module
- [x] Remove Snapshot optional non-audited module
- [x] Remove Killswitch
- [x] Remove OnlyDelegateCall security module
- [x] Remove MetaTx optional module
- [x] Remove Debt optional modules
- [x] Remove Validation optional module (neither rule engine, nor EIP1404 wrapper)

Close #AM-1320

### CODE REVIEW

**Comment by CarsonBrown96** [comment]
LGTM

**Review by CarsonBrown96 — APPROVED** [review]

File: truffle-config.js [hunk 1]
```diff
@@ -3,14 +3,14 @@ require('dotenv').config()
 module.exports = {
   networks: {
     ganache: {
-      host: "127.0.0.1",
-      port: 8545,
-      network_id: "*", // match any network
+      host: '127.0.0.1',
+      port: 7545,
+      network_id: 5777 // match any network
```
Comment (mariogutval) [inline truffle-config.js:9]:
```suggestion
      network_id: 5777
```

File: truffle-config.js [hunk 2]
```diff
@@ -3,14 +3,14 @@ require('dotenv').config()
...
     },
     live: {
-      host: "178.25.19.88", // Random IP for example purposes (do not use)
+      host: '178.25.19.88', // Random IP for example purposes (do not use)
       port: 80,
-      network_id: 1,        // Ethereum public network
+      network_id: 5777 // Ethereum public network
```
Comment (mariogutval) [inline truffle-config.js]:
```suggestion
      network_id: 5777
```

File: truffle-config.js [hunk 3]
```diff
@@ -28,7 +28,7 @@ module.exports = {
       settings: {
         optimizer: {
           enabled: true,
-          runs: 500
+          runs: 200
```
Comment (mariogutval) [inline truffle-config.js:38]: I think we should test several values to try to find the optimal one

**Review by mariogutval — CHANGES_REQUESTED** [review]

**Review by mariogutval — APPROVED** [review]

**Pull request merged**